uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
+        def staffers_by_job_options(self, job):
+            """
+            Format staffers_by_job() for use with the {% options %} template decorator
+            """
+            return [(a.id, a.full_name) for a in self.staffers_by_job(job)]
+
+        def staffers_by_job(self, job):
+            return self.query(Attendee)\
+                .filter_by(staffing=True)\
+                .filter(Attendee.assigned_depts.contains(str(job.location)))\
+                .filter(*(Attendee.trusted_depts.contains(str(job.location)),) if job.restricted else ())\
+                .order_by(Attendee.full_name).all()
+
         def search(self, text, *filters):
```

Comment:
one important note: after looking at it, it seems like ```session.staffers_by_job()``` and ```Job.available_staffers``` are pretty much identical.

only exception: ```Job.available_staffers``` invokes ```Job.no_overlap()```

in the future (not in this PR), it would probably be wise to see if we can just combine these two things together.

here's ```job.available_staffers```:

```python
    @cached_property
    def available_staffers(self):
        return [s for s in self.all_staffers
                if self.location in s.assigned_depts_ints
                   and (not self.restricted or s.trusted_in(self.location))
                   and self.no_overlap(s)]
```

if you guys think this is the case, we could add this as a github issue